docs/how-to/upgrade.md

```diff
@@ -26,6 +26,16 @@ If one is available, click the update button to update to the version indicated.
 !!! seealso alert alert-info "See also"
     [PMM Server Docker upgrade](../setting-up/server/docker.md#upgrade)
 
+## Updating a PMM-Agent
+
+PMM-Agent can be updated from tarball:
+
+- download tar.gz with pmm2-client
+- unpack it
+- run ./install_tarball script with "-u" flag.
+
+If you want update with rewriting config files run this script without any flags.
+
```

Comment:
Please find some minor changes:

PPMM-Agent can be updated from tarball:
		
PMM-Agent can be updated from tarball:
		
1. Download tar.gz with pmm2-client.
2. Extract it.
3. Run ./install_tarball script with the "-u" flag.

If you want to update the PMM-Agent with rewriting the config files, run this script without the "u" flag.

Reply:
The term "rewrite config file" will not be apparent to the user. Can we explain it more, so users know what and how this will affect them? What will be the difference between with flag and without a flag?

Reply:
I agree. 

We can have this as a hint or note:

Hint! The configuration file will be overwritten if you provide the "-u" flag while the PMM-Agent is updated.